types_webgl_wasm.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build js,wasm
+
+package gl
+
+import "syscall/js"
+
+type Enum int
+
+type Attrib struct {
```

Comment:
Question - would `type Attrib int` work? Using aliases instead of a singluar element struct may make some code simpler

Reply:
My understanding is that Attrib struct is used so that it allow the opengles port to use it as an interface and call function on it to do conversion for that port. As the goal is to make the same code work everywhere with gl-js (so you can try the web backend natively). I think moving to type Attrib int would mean a lot of code change with a lot of casting type around more than today. Which make me think it would not make the code simpler.

Reply:
Yeah fair point, I forgot that.